geometry_msgs/Point[] points
uint16 another_field